ops/ansible/playbooks-ccs/launch_bfd-pipeline.yml

```diff
@@ -4,8 +4,6 @@
 ##
 - name: Configure ETL Server
   hosts: localhost
-  vars:
-    env: "test"
```

Comment:
I can't think of any reason to to have a default `env`, potential vector for misconfiguration with agnostic AMIs.

Reply:
I feel like this was probably put in to avoid accidental deploys to other environments, but I have also have no issues with removing it.

Reply:
The scenario I was thinking of here was, if for some reason terraform doesn't properly pass the `env` var, I'd rather the deployment break entirely rather than just defaulting to test.

Reply:
I like this.